client/src/components/QuoteOfTheDay/QuoteService.test.js

```diff
@@ -0,0 +1,45 @@
+import {
+    fetchKanyeQuote,
+    fetchInspirationalQuote,
+    fetchTaylorSwiftQuote,
+    fetchProgrammingQuote,
+    fetchAnimeQuote,
+} from "./quoteService";
+
+function setText(text) {
+    return text;
+}
+
+function setAuthor(author) {
+    return author;
+}
+
+test("It fetches Kanye West quotes correctly", () => {
+    let { text, author } = fetchKanyeQuote(setText, setAuthor);
+    expect(text !== "");
+    expect(author === "Kanye West");
```

Comment:
I'm not 100% sure, but I think this would also work as

`expect(author).ToEqual("Kanye West")`

Same for the other test cases. However not a big deal, so not blocking merge

Reply:
@thakurtushar02 you're definitely right, using toEqual and other jest assertions is better for robust testing. I've just updated the tests, and included some extra assertions, would be great if you could have a look.
Thanks for the suggestion!

Reply:
Yep, just had a look at the new tests and happy with the changes!